packages/app-store-cli/src/build.ts

```diff
@@ -329,6 +328,14 @@ function generateFiles() {
       lazyImport: true,
     })
   );
+  browserOutput.push(
+    ...getExportedObject("EventTypeSettingsMap", {
+      importConfig: {
+        fileToBeImported: "components/EventTypeAppSettingsInterface.tsx",
+      },
+      lazyImport: true,
+    })
+  );
```

Comment:
create new mapping for EventTypeSettingsMap, need to create this component for all the apps that have EventTypeAppCardInterface.tsx